docs/_ext/autorobusta.py

```diff
@@ -253,7 +253,7 @@ def __generate_rst(
         trigger_params = json.loads(self.options.get("trigger-params", "{}"))
 
         example_yaml = generator.generate_example_config(
-            action_definition.func, recommended_trigger, trigger_params
```

Comment:
I don't think you should be removing this. It was added in [this PR](https://github.com/robusta-dev/robusta/pull/370/files) and removing it breaks one of the features added there.

Perhaps you need to run `poetry install` to get it working? Maybe for some reason the docs are importing an older version of the robusta library which doesn't have this?

Reply:
could be ill check this